add VS Code instructions to README (#62)
apify/actors-mcp-server: README.md

For one-click installation, click one of the install buttons below:

[![Install with NPX in VS Code](https://img.shields.io/badge/VS_Code-NPM-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=actors-mcp-server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40apify%2Factors-mcp-server%22%5D%2C%22env%22%3A%7B%22APIFY_TOKEN%22%3A%22%24%7Binput%3Aapify_token%7D%22%7D%7D&inputs=%5B%7B%22type%22%3A%22promptString%22%2C%22id%22%3A%22apify_token%22%2C%22description%22%3A%22Apify+API+Token%22%2C%22password%22%3Atrue%7D%5D) [![Install with NPX in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-NPM-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=actors-mcp-server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40apify%2Factors-mcp-server%22%5D%2C%22env%22%3A%7B%22APIFY_TOKEN%22%3A%22%24%7Binput%3Aapify_token%7D%22%7D%7D&inputs=%5B%7B%22type%22%3A%22promptString%22%2C%22id%22%3A%22apify_token%22%2C%22description%22%3A%22Apify+API+Token%22%2C%22password%22%3Atrue%7D%5D&quality=insiders)

> ### Manual Installation

Reviewer: ```suggestion
##### Manual installation
```

You can manually install the Apify MCP Server in VS Code. First, click one of the install buttons at the top of this section for a one-click installation.

Alternatively, add the following JSON block to your User Settings (JSON) file in VS Code. You can do this by pressing `Ctrl + Shift + P` and typing `Preferences: Open User Settings (JSON)`.

```json
{
  "mcp": {
    "inputs": [
      {
        "type": "promptString",
        "id": "apify_token",
        "description": "Apify API Token",
        "password": true
      }
    ],
    "servers": {
      "actors-mcp-server": {
        "command": "npx",
        "args": ["-y", "@apify/actors-mcp-server"],
        "env": {
          "APIFY_TOKEN": "${input:apify_token}"
        }
      }
    }
  }
}
```

> Optionally, you can add it to a file called `.vscode/mcp.json` in your workspace. This will allow you to share the configuration with others.
> 
> ```json
> {
>   "inputs": [
>     {
>       "type": "promptString",
>       "id": "apify_token",
>       "description": "Apify API Token",
>       "password": true
>     }
>   ],
>   "servers": {
>     "actors-mcp-server": {
>       "command": "npx",
>       "args": ["-y", "@apify/actors-mcp-server"],
>       "env": {
>         "APIFY_TOKEN": "${input:apify_token}"
>       }
>     }
>   }
> }
> ```

Reviewer: ```suggestion
Optionally, you also add the above configuration into file called `.vscode/mcp.json` in your workspace. This will allow you to share the configuration with others.
```

Reviewer: The Readme file is becoming quite long, Perhaps we can omit one JSON example as they are the same?

Author: @jirispilka - Agreed. Lets do that and just let  them know to omit the `mcp{}` key if they put it in the .vscode/mcp.json file. Cutting out the last JSON blob example should cut it down a good amount.

Author: Also the changes you made look good!

> Optionally, you can add it to a file called `.vscode/mcp.json` in your workspace. This will allow you to share the configuration with others.
> 
> ```json
> {
>   "inputs": [
>     {
>       "type": "promptString",
>       "id": "apify_token",
>       "description": "Apify API Token",
>       "password": true
>     }
>   ],
>   "servers": {
>     "actors-mcp-server": {
>       "command": "npx",
>       "args": ["-y", "@apify/actors-mcp-server"],
>       "env": {
>         "APIFY_TOKEN": "${input:apify_token}"
>       }
>     }
>   }
> }
> ```

Reviewer: ```suggestion
Optionally, you can add it to a file called `.vscode/mcp.json` in your workspace - just omit the top-level `mcp {}` key. This will allow you to share the configuration with others.
```
